### ISSUE TITLE
Wouldn't it be nice...

### ISSUE THREAD

**Issue by peschkaj** [legitbiz/SealsOfFate#54]
- [ ] if armor was called blubber and armor pick ups were a blubber booster?
- [ ] we had [pinniped](https://en.wikipedia.org/wiki/Pinniped) power?
- [ ] if there was a level called "snow bonk"?

**Comment by connorkuehl** [comment]
I think there are diminishing returns on blubber armor because at a certain point the blubber mass index suggests a bleak morbid obesity :-( That said, I think it's a great idea. 

What? :joy:

Must-have.

**Comment by peschkaj** [comment]
Agreed - I've gone and added the armor blubber to the combat issue (#56).

**Comment by Zonr0** [comment]
In the spirit of our caffeine addicted penguins that seem to always be finding new stashes of ~~cocaine~~ caffeine, we should have an over-caffeinated enemy prefix. Every turn an enemy with this prefix would have a small chance to switch to the TweakingOut state, which dramatically increases their speed but causes them to move and act randomly.
